apps/camera/js/lib/geo-location.js

```diff
@@ -39,7 +33,7 @@ function GeoLocation() {
  */
 GeoLocation.prototype.watch = function() {
   if (!this.watcher) {
-    this.watcher = geolocation.watchPosition(this.setPosition);
+    this.watcher = navigator.geolocation.watchPosition(this.setPosition);
```

Comment:
Note that the mere act of accessing navigator.geolocation creates the geolocation object along with underlying services, etc. So getting rid of the local above took that creation off the critical path.